classes/controller/menu/class-archive-import.php

```diff
@@ -0,0 +1,250 @@
+<?php
+/**
+ * An importer for archived content from an XML file.
+ *
+ * @package P4GBKS\Controllers\Menu
+ */
+
+namespace P4GBKS\Controllers\Menu;
+
+use WP_Query;
+
+/**
+ * An importer for archived content from an XML file.
+ *
+ * @package P4GBKS\Controllers\Menu
+ */
+class Archive_Import extends Controller {
+
+	private const WAY_BACK_URL = 'https://wayback.archive-it.org/9650/';
+
+	private const P3_BASE_URL = 'http://p3-raw.greenpeace.org/';
```

Comment:
The fixed p3 link has a excuse for NL(https://github.com/greenpeace/planet4-gpi-export/blob/master/P3%20Archive%20Elastic%20Search%20data/data/netherlands/netherlands-nl_post_list_with_date_final-clean.csv)
We need to handle it with some hardcoded condition.

Reply:
Are you ok with taking this up later? It's only for a couple of NROs, so we can start with those where this is not the case.